### DISCUSSION TITLE
[v3] Documentation <-> code mismatch: dynamic components

### DISCUSSION THREAD

**Discussion by levu42** [livewire/livewire#6482] [1. Help]
According to the documentation, I should be able to do:

```
<livewire:dynamic-component :is="$componentName" />
```

but that leads to this exception: `ComponentAttributeMissingOnDynamicComponentException` ("Dynamic component tag is missing component attribute.") in [CompileLivewireTags:74](https://github.com/livewire/livewire/blob/main/src/Mechanisms/CompileLivewireTags.php#L74)

Replacing `:is` with `:component` works instead, but it's not the documented way to do it.

Since I'm not sure, whether the documented or the implemented way is the correct one, I won't submit a PR for it.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joshhanley** [comment]
@levu42 there was a bug which has been fixed today and will be in the next release 🙂
